### PR TITLE
fix: header counter position

### PR DESCRIPTION
**Pull request by tnrdd**
# Description

Center the position of the funds raised counter in the header.

# Checklist:

- [x] My commit message follows the Conventional Commits specification
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have tested my code
- [x] My changes generate no new warnings
- [x] My PR is rebased off the most recent `develop` branch
- [x] My PR is opened against the `develop` branch

# Alert Reviewers

@codynhat @gravenp

### CODE REVIEW

File: src/components/shared/FundsRaisedCounter.tsx [hunk 1]
```diff
@@ -27,7 +27,8 @@ function FundsRaisedCounter() {
   );
   const accountTokenSnapshot = data?.items[0];
   const amount = useFlowingAmount(
-    BigInt(accountTokenSnapshot?.totalAmountStreamedInUntilUpdatedAt ?? 0) +
```
Comment (gravenp) [inline src/components/shared/FundsRaisedCounter.tsx:30]: @tnrdd Was this change intentional? I think it takes us back to understating the amount raised.

Comment (tnrdd) [reply src/components/shared/FundsRaisedCounter.tsx:30]: Using only `totalAmountStreamedUntilUpdatedAt` is not enough because it wouldn't count the 0.005 received on parcels claim, `balanceUntilUpdatedAt + totalAmountStreamedOutUntilUpdatedAt + totalAmountTransferredUntilUpdateAt` should be the correct value.

Comment (gravenp) [reply src/components/shared/FundsRaisedCounter.tsx:30]: Got it. On the same page now. TY!

**Review by gravenp — APPROVED** [review]
TY!
